docs/spec/broker.md

```diff
@@ -44,6 +44,14 @@ The attributes filter specifying a list of key-value pairs MUST be supported by
 Trigger. Events that pass the attributes filter MUST include context or
 extension attributes that match all key-value pairs exactly.
 
+### Delivery Spec
+
+Both BrokerSpec and TriggerSpec have a `Delivery` field of type `duck.DeliverySpec` (per `v1`).
+This field allows the user to define the dead letter sink and retries.
+The `BrokerSpec.Delivery` field is global across all the Triggers registered with that particular
+Broker, while the `TriggerSpec.Delivery`, if configured, fully overrides `BrokerSpec.Delivery` for
+that particular Trigger.
```

Comment:
This doesn't exist: https://github.com/knative/eventing/blob/1e06e5ce4c64d0f31e7f687fff942493e35fe060/pkg/apis/eventing/v1/trigger_types.go#L74-L87

Reply:
With this PR it does https://github.com/knative/eventing/pull/4654/files#diff-9897c649cc6cb820cddfd787a041f960074f91d38b87bd24216e110e76718c1eR92

Reply:
Do we really need it? 
There are no feature requests for this, so I'd keep it as it is for now.
Basically, this comment: https://github.com/knative/eventing/issues/4515#issuecomment-726297847

Reply:
I don't know, for me we should have it for consistency and also because it's trivial to implement (basically, this can be implemented at the duck type controller, like for channel https://github.com/knative/eventing/pull/4652)

Reply:
I totally see your point on consistency, but Individual implementations manage triggers themself, there is no generic thing like that for triggers, so it requires a little bit more work. 
For that reason and for the reason in my previous comment, I think we can defer this decision at some later point in time, and IMO clarifying the spec and adding a field should be 2 different PRs since one clarifies what we already have and the other one introduces a new feature.

Reply:
We can bring it up in the delivery WG meeting today.

Reply:
As discussed during today delivery WG meeting, we're going to add this